### PR TITLE
Added Color Transfer node

### PR DESCRIPTION
**Pull request by theflyingzamboni**

### CODE REVIEW

File: backend/nodes/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (RunDevelopment) [inline backend/nodes/LICENSE.txt:1]: This isn't how you typically include a license in open source. It's not what source is licensed here, so the license effectively applies to everything in this directory.

It seems like [this](https://stackoverflow.com/questions/1523427/what-is-the-common-header-format-of-python-files) is common practice in python, so we should probably follow it. Just move the licensed function into its own file.

Comment (joeyballentine) [reply backend/nodes/LICENSE.txt:1]: Yeah I definitely don't want to accidentally change the license of some of my files

File: backend/nodes/image_nodes.py [hunk 2]
```diff
@@ -1508,130 +1509,106 @@ def run(self, img: np.ndarray, method: int | str) -> np.ndarray:
...
+                ],
+            ),
+            DropDownInput(
+                "generic",
+                "Reciprocal Scaling Factor",
+                [
+                    {"option": "Yes", "value": 1},
+                    {"option": "No", "value": 0},
+                ],
+            ),
```
Comment (RunDevelopment) [inline backend/nodes/image_nodes.py:1571]: Do we not have a checkbox input?

File: backend/nodes/image_nodes.py [hunk 3]
```diff
@@ -1508,130 +1509,106 @@ def run(self, img: np.ndarray, method: int | str) -> np.ndarray:
...
-            blurred = self.fragment_blur(img, 5, i, 1 << i)
-            # blurred tends to be a bit too transparent
-            self.with_self_as_background(blurred)
-            with_background(result, blurred)
+    def __init__(self):
+        """Constructor"""
+        super().__init__()
+        self.description = "Transfers colors from reference image"
+        self.inputs = [
+            ImageInput("Image"),
```
Comment (RunDevelopment) [inline backend/nodes/image_nodes.py:1545]: Isn't "Image" the default name?

File: backend/nodes/image_nodes.py (hunk 2 above)
Comment (joeyballentine) [reply backend/nodes/image_nodes.py:1571]: Nope, haven't needed it yet

File: backend/nodes/image_nodes.py (hunk 3 above)
Comment (joeyballentine) [reply backend/nodes/image_nodes.py:1545]: It is, I'm pretty sure

**Pull request merged**